content_server.py

```diff
@@ -22,25 +22,30 @@ class LookupClientCoverageProvider(CatalogCoverageProvider):
     """
 
     # TODO: We should rename this because in theory it can be used
-    # other places, but in practice this is it.
+    # other places, but in practice this is it. If this ever changes in
+    # practice, COVERAGE_COUNTS_FOR_EVERY_COLLECTION should also be set to
+    # False.
     SERVICE_NAME = "OA Content Server Coverage Provider"
+    COVERAGE_COUNTS_FOR_EVERY_COLLECTION = True
     PROTOCOL = ExternalIntegration.OPDS_IMPORT
-    
+
     OPDS_SERVER_RETURNED_WRONG_CONTENT_TYPE = "OPDS Server served unhandleable media type: %s"
    
     def __init__(self, collection, **kwargs):
         self.DATA_SOURCE_NAME = collection.data_source.name
         super(LookupClientCoverageProvider, self).__init__(collection, **kwargs)
 
+        # The feed URL may be a partial list. Get the root URL.
+        feed_url = collection.external_account_id
+        base_url = urlparse.urljoin(feed_url, '/')
+
```

Comment:
When would this happen?

Reply:
In the case of instant classics collections, where the external_account_id is set to "http://oacontent.librarysimplified.org/works/lists/Instant%20Classics".

Reply:
Got it.